<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0xa0 in position 18363: invalid start byte http://www.ics.uci.edu/~hshirani/webobjects
<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0xa0 in position 91244: invalid start byte http://www.ics.uci.edu/~cs224
<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0x92 in position 22754: invalid start byte http://www.ics.uci.edu/%7Edsm/compose/compose_design.html
<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0x92 in position 2502: invalid start byte http://esl.ics.uci.edu/research.html
<class 'ValueError'> Unicode strings with encoding declaration are not supported. Please use bytes input or XML fragments without declaration. http://www.ics.uci.edu/~jajones
<class 'ValueError'> Unicode strings with encoding declaration are not supported. Please use bytes input or XML fragments without declaration. http://www.ics.uci.edu/~gmark
<class 'ValueError'> Unicode strings with encoding declaration are not supported. Please use bytes input or XML fragments without declaration. http://www.ics.uci.edu/~djr
<class 'ValueError'> Unicode strings with encoding declaration are not supported. Please use bytes input or XML fragments without declaration. http://www.ics.uci.edu/~taylor
<class 'ValueError'> Unicode strings with encoding declaration are not supported. Please use bytes input or XML fragments without declaration. http://www.ics.uci.edu/~emilyo/SimSE
<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0xad in position 2991: invalid start byte http://www.ics.uci.edu/~redmiles/ics280-WQ02
<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0xad in position 3923: invalid start byte http://www.ics.uci.edu/~redmiles/ics229-SQ03
<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0x92 in position 18022: invalid start byte http://www.ics.uci.edu/~redmiles/ics131-FQ03
<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0xa0 in position 18786: invalid start byte http://www.ics.uci.edu/~redmiles/ics221-FQ03
<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0x93 in position 16904: invalid start byte http://www.ics.uci.edu/~redmiles/ics227-SQ04
<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0xa0 in position 8681: invalid start byte http://www.ics.uci.edu/~redmiles/ics131-FQ04
<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0xa0 in position 13148: invalid start byte http://www.ics.uci.edu/~redmiles/ics203b-SQ05
<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0xa0 in position 13198: invalid start byte http://www.ics.uci.edu/~redmiles/ics205-FQ05
<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0xf6 in position 14423: invalid start byte http://www.ics.uci.edu/~redmiles/ics229-WQ06
<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0xa0 in position 12124: invalid start byte http://www.ics.uci.edu/~redmiles/ics104-WQ06
<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0xa0 in position 11986: invalid start byte http://www.ics.uci.edu/~redmiles/inf44-SQ07
<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0xa0 in position 10180: invalid start byte http://www.ics.uci.edu/~redmiles/inf233-FQ07
<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0xa0 in position 11575: invalid start byte http://www.ics.uci.edu/~redmiles/inf143-SQ08
<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0xa0 in position 11383: invalid start byte http://www.ics.uci.edu/~redmiles/inf143-SQ09
<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0xa9 in position 3380: invalid start byte http://hobbes.ics.uci.edu
<class 'ValueError'> Unicode strings with encoding declaration are not supported. Please use bytes input or XML fragments without declaration. http://www.ics.uci.edu/~franklin
<class 'ValueError'> Unicode strings with encoding declaration are not supported. Please use bytes input or XML fragments without declaration. https://www.ics.uci.edu/~eppstein
<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0xb7 in position 26814: invalid start byte http://www.ics.uci.edu/~dchudova
<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0x9a in position 2721: invalid start byte http://computableplant.ics.uci.edu/papers/2004/BGRS-2004-KAP-abstract.html
<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0x96 in position 29292: invalid start byte http://computableplant.ics.uci.edu/papers
<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0xff in position 0: invalid start byte http://www.ics.uci.edu/%7Emajumder/pub.html
<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0x97 in position 77535: invalid start byte http://www.ics.uci.edu/~gopi/Resume.html
<class 'ValueError'> Unicode strings with encoding declaration are not supported. Please use bytes input or XML fragments without declaration. http://www.ics.uci.edu/~djr/DebraJRichardson/Home.html
<class 'ValueError'> Unicode strings with encoding declaration are not supported. Please use bytes input or XML fragments without declaration. http://www.ics.uci.edu/~gmark/Home_page/Welcome.html
<class 'ValueError'> Unicode strings with encoding declaration are not supported. Please use bytes input or XML fragments without declaration. http://www.ics.uci.edu/~thornton
<class 'ValueError'> Unicode strings with encoding declaration are not supported. Please use bytes input or XML fragments without declaration. http://www.ics.uci.edu/~ajfrank
<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0x97 in position 37104: invalid start byte http://sli.ics.uci.edu/Pubs/Abstracts
<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0xf9 in position 4458: invalid start byte http://sli.ics.uci.edu/Site/Search
<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0xff in position 0: invalid start byte http://www.ics.uci.edu/~dutt
<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0xfc in position 14312: invalid start byte http://www.ics.uci.edu/~jutts
<class 'ValueError'> Unicode strings with encoding declaration are not supported. Please use bytes input or XML fragments without declaration. http://www.ics.uci.edu/~dgillen/Welcome.html
<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0xa0 in position 45567: invalid start byte https://www.ics.uci.edu/~wjohnson
<class 'KeyError'> 'content-type' http://www.ics.uci.edu/~yamingy/npmle.r
<class 'KeyError'> 'content-type' http://www.ics.uci.edu/~yamingy/bayesD_demo3.r
<class 'KeyError'> 'content-type' http://www.ics.uci.edu/~yamingy/bayesD_demo2.r
<class 'KeyError'> 'content-type' http://www.ics.uci.edu/~yamingy/bayesD_demo1.r
<class 'KeyError'> 'content-type' http://www.ics.uci.edu/~yamingy/bayesD_cocktail.r
<class 'KeyError'> 'content-type' http://www.ics.uci.edu/~yamingy/cocktail_demo4.r
<class 'KeyError'> 'content-type' http://www.ics.uci.edu/~yamingy/cocktail_demo3.r
<class 'KeyError'> 'content-type' http://www.ics.uci.edu/~yamingy/cocktail_demo2.r
<class 'KeyError'> 'content-type' http://www.ics.uci.edu/~yamingy/cocktail_demo1.r
<class 'KeyError'> 'content-type' http://www.ics.uci.edu/~yamingy/cocktail.r
<class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0x96 in position 26885: invalid start byte http://www.ics.uci.edu/~isaac
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11b8af9e8>: Failed to establish a new connection: [Errno 60] Operation timed out')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=majumder
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11b8bc208>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=lueker
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11b8bc4a8>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=chenli
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11b8bc0f0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=levorato
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=http%3A%2F%2Fwww.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11b8bcf28>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) http://www.ics.uci.edu/~mlevorat/ 
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11b8bc4a8>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=rickl
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11b8bcef0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=klefstad
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11b8bc588>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=dfkibler
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=http%3A%2F%2Fwww.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11b8bcfd0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) http://www.ics.uci.edu/~kibler
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11b8bca90>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=kay
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11b8bc0f0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=kkask
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c550>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=sabdujyo
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=http%3A%2F%2Fwww.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11b8bc550>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) http://www.ics.uci.edu/~sabdujyo
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c160>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=swjun
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cc88>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=sjordan
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c828>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=sjarecki
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=http%3A%2F%2Fwww.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c898>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) http://www.ics.uci.edu/~stasio
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c160>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=rcjain
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c828>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=ssirani
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c898>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=aihler
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11b8bca90>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=mustafai
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11b8bca90>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.ics.uci.edu/~mustafai
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11b8bca90>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=dhirschb
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11b8bca90>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=whayes
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=http%3A%2F%2Fwww.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11b8bca90>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) http://www.ics.uci.edu/~wayne
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11b8bc0f0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=iharris
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11b8bc4a8>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=goodrich
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c710>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=givargis
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c550>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=igassko
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c358>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=sgagomas
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cc88>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=franz
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=http%3A%2F%2Fwww.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c710>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) http://www.ics.uci.edu/~franz
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c588>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=royf
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c358>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=fowlkes
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c9b0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=eppstein
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c160>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=elzarki
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cef0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=dutt
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c828>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=mbdillen
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=http%3A%2F%2Fwww.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c390>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) http://www.ics.uci.edu/~dillenco
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c358>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=bdemsky
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=http%3A%2F%2Fplrg.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c208>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) http://plrg.ics.uci.edu
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cc88>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=dechter
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c358>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=alfchen
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c898>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=mjcarey
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c390>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=aburtsev
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c160>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=ebozorgz
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=http%3A%2F%2Fwww.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c828>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) http://www.ics.uci.edu/~eli
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cfd0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=lbic
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=http%3A%2F%2Fwww.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c710>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) http://www.ics.uci.edu/~bic
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c898>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=pfbaldi
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c828>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=ardalan
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c2b0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty/profiles/view_faculty.php?ucinetid=alfaro
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=http%3A%2F%2Fwww.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c9b0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) http://www.ics.uci.edu/~stauro
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cc88>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/staff
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c2b0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/slider/zybooks-touts-less-text-more-action
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c8d0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/slider/unlocking-the-potential-of-deep-learning
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c208>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/slider/uci-team-develops-large-scale-social-media-analysis-system
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c780>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/slider/uci-team-awarded-2-9-million-by-nsf-for-graduate-training-in-data-science-stem
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c2e8>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/slider/uci-cybersecurity-policy-research-institute-launches-initiatives-to-combat-cyber-threats
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cc88>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/slider/typing-while-skyping-could-compromise-privacy
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cf28>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/slider/two-ics-professors-named-2015-acm-fellows
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c390>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/slider/robots-to-the-rescue
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cef0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/slider/putting-theory-into-practice
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c2b0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/slider/professor-michael-carey-named-ieee-fellow
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c390>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/slider/professor-franz-amasses-3-9-million-in-research-funding
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11b8b2d68>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/slider/new-professorship-oracle-gift-for-franz
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c400>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/slider/new-dean-named-for-ics
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c710>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/slider/mehrotra-set-to-lead-two-nsf-funded-research-projects
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c550>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/slider/levorato-receives-prestigious-hellman-fellowship
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c080>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/slider/icssc-presents-google-themed-hackathon
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cf98>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/slider/ics-welcomes-four-new-faculty-members-for-fall-2016-quarter
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9ccf8>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/slider/ics-alumnus-named-to-oc-registers-100-most-influential-list
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c8d0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/slider/how-vulnerable-are-the-nations-electronic-voting-machines
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c390>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/slider/franz-looks-to-simplify-software-for-increased-security
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c550>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/slider/eppstein-named-aaas-fellow
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c080>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/slider/ant
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cf98>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/slider/cs-undergrads-business-makes-inc-s-2016-list-of-coolest-college-startups
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9ccf8>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/slider/carey-receives-2016-ieee-tcde-csee-impact-award
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c160>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/slider/bringing-cybersecurity-to-the-students
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cc88>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/slider/163-2
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c828>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/slider/amiri-sani-received-1-million-from-nsf-in-2016
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c8d0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/slider/3-ics-researchers-named-aaas-fellows
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11b8b2588>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/slider
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11b8b2588>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/seminar-series-winter-2017
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11b8b2588>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/seminar-series-use-full-width-template
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c780>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/seminar-series-testing
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c400>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/seminar-series-archive
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cf98>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/sample-page
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c2e8>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/protecting-mobile-users-behind-the-scenes
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c898>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/professor-xu-awarded-2018-aito-junior-dahl-nygaard-prize
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c208>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/professional-master-of-computer-science
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c2b0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/homepage-etch
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c8d0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/csgradvisitday2019
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cf98>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/feed-test
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c2e8>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty-list-test
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c898>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty-backup
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c208>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/faculty-test
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cc88>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/events-filter-test
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c390>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/events-page/computer-science-seminar-series
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c080>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/events-page
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c2b0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/events/seminar-series-testing
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c390>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/events/seminar-series/seminar-series-spring-2017
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11b8b2780>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/events/seminar-series/seminar-series-fall-2016
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11b8b2780>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/events/seminar-series/csnetsys-seminar-series-prof-k-k-ramakrishnan-university-of-california-riverside-software-based-networks-leveraging-high-performance-nfv-platforms-to-meet-future-communicat
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c9b0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/events/seminar-series/cs-seminar-series-roman-lysecky-university-of-arizona-runtime-anomaly-detection-rad
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c780>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/events/seminar-series/cs-seminar-series-radu-marculescu-cmu-entering-the-labyrinth-of-inexact-computer-science-a-cyber-physical-systems-approach
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cc88>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/events/seminar-series/cs-seminar-series-prof-wen-gao-peking-university-enabling-data-sharing-with-secure-computation
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c080>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/events/seminar-series/cs-seminar-series-prof-sameer-singh-uc-irvine-towards-intuitive-explanations-and-interactions-with-black-box-machine-learning
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cda0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/events/seminar-series/cs-seminar-series-prof-ruslan-salakhutdinov-cmu-learning-deep-unsupervised-and-multimodal-models
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c278>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/events/seminar-series/cs-seminar-series-prof-roger-zimmermann-national-university-of-singapore-dash-streaming-with-software-defined-networking-support
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c208>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/events/seminar-series/cs-seminar-series-prof-majid-sarrafzadeh-uiuc-saving-lives-one-byte-at-a-time
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c828>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/events/seminar-series/cs-seminar-series-prof-ashwin-machanavajjhala-duke-university-utility-cost-of-provable-privacy-a-case-study-on-us-census-bureau-data
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cc88>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/events/seminar-series/cs-seminar-series-dr-kathleen-knobe-rice-university-cnc-a-dependence-programming-language
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c080>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/events/seminar-series/cs-seminar-series-dr-kai-berger-caltech-recent-developments-in-polarization-approaches-to-computer-vision
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cda0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/events/seminar-series/cs-seminar-series-dr-barath-raghavan-icsi-berkeley-frontier-networks-context-challenges-and-connectivity
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c278>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/events/seminar-series/cs-seminar-series-barbara-j-grosz-harvard-university-from-the-turing-test-to-smart-partners-is-your-system-smart-enough-to-work-with-us
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c898>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/events/seminar-series/cs-seminar-series-sept-23-2016-zvonimir-rakamaric-university-of-utah
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c2b0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/events/seminar-series/cs-seminar-series-prof-anton-burtsev-university-of-utah-lightweight-capability-domains-towards-secure-operating-system-kernels
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cfd0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/events/distinguished-lecture-series/distinguished-lecture-series-2017
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c208>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/events/research-showcase
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11b8b2d68>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/events
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11b8b2d68>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/distinguished-lectures-series-archived-event-listing
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11b8b2d68>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/csnetsys-seminar-series-prof-zhiyun-qian-uc-riverside-off-path-tcp-exploits-an-oversight-yesterday-a-lingering-threat-today
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c550>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/csnetsys-seminar-series-prof-tam-vu-university-of-colorado-denver-enabling-data-sharing-with-secure-computation
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c9b0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/cs-seminar-series-prof-sanjay-k-jha-unsw-a-changing-landscape-securing-the-internet-of-things-iot
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cda0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/csnetsys-seminar-series-dr-vlad-kolesnikov-bell-labs-enabling-data-sharing-with-secure-computation
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c160>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/cs-seminar-series-sushant-jain-google-large-scale-and-low-latency-data-distribution-from-database-to-servers
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cf98>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/cs-seminar-series-prof-xuehai-qian-university-of-southern-california-breaking-the-myth-of-think-as-a-vertex
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c588>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/cs-seminar-series-prof-qi-alfred-chen-uc-irvine-ghost-cars-and-fake-obstacles-automated-security-analysis-of-self-driving-car-and-smart-traffic-light-systems
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cf28>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/filter-test
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c208>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/chairs-welcome
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cda0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/art-and-computers-symposium/video-images
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c160>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/art-and-computers-symposium/sponsors
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cf98>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/art-and-computers-symposium/acsymposium-speakers
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c588>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/art-and-computers-symposium/program
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c2b0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/art-and-computers-symposium/acsymposium-organizers
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c828>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/art-and-computers-symposium/acsymposium-location
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c080>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/art-and-computers-symposium
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cf28>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://www.cs.uci.edu/alumni-success-story-ersin-uzun
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fgrape.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c828>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://grape.ics.uci.edu/wiki/public/wiki/cs122b-2012-spring
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fgrape.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11b8b2780>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://grape.ics.uci.edu/wiki/public/wiki/cs222-2012-fall
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fgrape.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c2e8>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://grape.ics.uci.edu/wiki/public/wiki/cs122b-2013-winter
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=http%3A%2F%2Fflamingo.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c828>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) http://flamingo.ics.uci.edu
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=http%3A%2F%2Fwww.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c828>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) http://www.ics.uci.edu/community/news/notes/index
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=http%3A%2F%2Fasterixdb.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cf98>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) http://asterixdb.ics.uci.edu/documentation/index.html
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fgrape.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c2e8>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://grape.ics.uci.edu/wiki/public/wiki/cs222-2015-fall
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fgrape.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c828>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://grape.ics.uci.edu/wiki/public/wiki/cs122b-2016-winter
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fgrape.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c780>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://grape.ics.uci.edu/wiki/public/wiki/cs122a-2016-spring
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=http%3A%2F%2Fwww.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cb38>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) http://www.ics.uci.edu/~cs222
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=http%3A%2F%2Fcloudberry.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c278>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) http://cloudberry.ics.uci.edu
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fgrape.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c9b0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://grape.ics.uci.edu/wiki/public/wiki/cs122b-2017-winter
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fgrape.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cb38>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://grape.ics.uci.edu/wiki/public/wiki/cs122b-2017-spring
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=http%3A%2F%2Fwww.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c2e8>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) http://www.ics.uci.edu/~jianfenj
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=http%3A%2F%2Fcloudberry.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c400>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) http://cloudberry.ics.uci.edu/demos/twittermap
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fgrape.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cb38>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://grape.ics.uci.edu/wiki/public/wiki/cs122b-2018-winter
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=http%3A%2F%2Fwww.cs.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11afe7828>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) http://www.cs.uci.edu/socal-social-analytics-workshop-serves-as-catalyst-for-multidisciplinary-collaborations
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fgrape.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c780>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://grape.ics.uci.edu/wiki/public/wiki/cs222p-2018-fall
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fgrape.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c780>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://grape.ics.uci.edu/wiki/public/wiki/cs222-2018-fall
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fgrape.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c8d0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://grape.ics.uci.edu/wiki/public/wiki/cs122b-2019-winter
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fgrape.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c710>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://grape.ics.uci.edu/wiki/public/wiki/cs221-2019-spring
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fchenli.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c780>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://chenli.ics.uci.edu/student
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fchenli.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9ccf8>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://chenli.ics.uci.edu/teaching
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fchenli.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cda0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://chenli.ics.uci.edu/students
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fchenli.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9cef0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://chenli.ics.uci.edu/publications
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fchenli.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c2e8>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://chenli.ics.uci.edu/research
<class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='styx.ics.uci.edu', port=9004): Max retries exceeded with url: /?q=https%3A%2F%2Fchenli.ics.uci.edu%2Frobots.txt&u=IR+F19+38314561%2C22347485%2C17424317 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11af9c710>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')) https://chenli.ics.uci.edu/biography
